Formula/afsctool.rb

```diff
@@ -17,46 +16,25 @@ class Afsctool < Formula
     sha256 cellar: :any_skip_relocation, sierra:        "96437b04a2974c215979550d3d70b4c8e3f609e76954ca41059c6f246da452ee"
   end
 
-  depends_on :macos
-
-  # Fixes Sierra "Unable to compress" issue; reported upstream on 24 July 2017
-  patch :p2 do
-    url "https://github.com/vfx01j/afsctool/commit/26293a3809c9ad1db5f9bff9dffaefb8e201a089.patch?full_index=1"
-    sha256 "69ec72b2d6f89b53e93c7bacef1916ea4cf815e4b3e7ab4ee8010c31de1d4e66"
-  end
-
-  # Fixes High Sierra "Expecting f_type of 17 or 23. f_type is 24" issue
-  # Acknowledged by upstream 12 Apr 2018:
-  # https://github.com/Homebrew/homebrew-core/pull/20898#issuecomment-380727547
-  patch :p2, :DATA
+  depends_on "cmake" => :build
+  depends_on "google-sparsehash" => :build
+  depends_on "pkg-config" => :build
+  uses_from_macos "zlib", since: :monterey
```

Comment:
Is there a reason for the `zlib` to be only since Monterey?  It's unusual for us to have make this type of caveat for something as widely used `zlib`.

Also please add `depends_on :macos` if this should not be built on Linux.  It appears right now that build could succeed if we added `libbsd` as a dependency, but it's not clear to me if it this tool is actually useful on Linux.

Reply:
Sorry I missed your above comment.  You don't need to add `depends_on :macos`.  I see that this also works with ZFS.  I will try to build this in Docker on Linux to see if anything other than `libbsd` is needed.

Reply:
`zlib`: You are right. I got confused by the output of the CI here: https://github.com/Homebrew/homebrew-core/actions/runs/1899328130

`libbsd`: yep that seems to be the only thing needed https://github.com/Homebrew/homebrew-core/runs/5402676843?check_suite_focus=true